templates/scripts.tmpl/cleanup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Please run this script only in the project root folder.
+#
+# Usage
+# npm run clean
+#
+# To delete node_modules folder:
+# npm run clean -- all
+
+rm -r dist/
+rm -r dist-node/
+rm -r dist-browser/
+rm -r browsertest.build/
+rm -r .nyc_output/
+rm -r *.log
```

Comment:
I'm worried this will delete logs people might want to keep. Is there anything besides `npm-debug.log` and `lerna-debug.log`? Also I think npm debug logs can end with timestamps or something sometimes.